src/components/molecules/NavSearchBar/NavSearchResult.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+import { UserAvatar } from "components/atoms/UserAvatar";
+
+import "./NavSearchResult.scss";
+
+export interface NavSearchResultProps {
+  title: string;
+  description?: string;
+  image?: string;
+  isAvatar?: boolean;
+  onClick?: (e?: React.MouseEvent) => void;
+}
+
+export const NavSearchResult: React.FC<NavSearchResultProps> = ({
+  title,
+  description,
+  image,
+  isAvatar,
+  onClick,
+}) => {
+  const imageStyles = {
+    backgroundImage: image ? `url(${image})` : undefined,
+  };
+
+  return (
+    <div className="NavSearchResult font-size--small" onClick={onClick}>
+      {isAvatar ? (
+        <UserAvatar avatarSrc={image} />
+      ) : (
+        <div className="NavSearchResult__image" style={imageStyles} />
+      )}
+
+      <div>
```

Comment:
Do we need this extra `div` here? Can we remove it?

Reply:
we need it for correct `display: flex` applying.

```html
<div styles="<flex, flex-direction: row>" >
    <div>{icon}</div>

    <div>
        <div>{title}</div>
        <div>{description}</div>
    </div>
</div>
```

Reply:
[NB] Makes sense. I wonder if we should add a comment/similar there describing this? Or potentially give it a BEM class (`&__content`?) or similar (even though we don't have any styles to add to it) just to make its purpose a bit more 'semantic'?

Reply:
added the class name `NavSearchResult__content`: https://github.com/sparkletown/sparkle/pull/1328/commits/5afa888773470274a321c8b6e1ba1e547b72776c